dtfs-central-api/src/repositories/tfm-facilities-repo/tfm-facility.schema.ts

```diff
@@ -2,9 +2,8 @@ import z from 'zod';
 import { MongoObjectIdSchema } from '../../v1/routes/middleware/payload-validation/schemas';
 
 const TfmFacilityDateFieldSchema = z
```

Comment:
Little bit worried about this if it's not tested for `coverEndDate` -- Could we get some tests testing for all methods of how we store date in our database plz? :)

Reply:
See [this comment](https://github.com/UK-Export-Finance/dtfs2/pull/3331#discussion_r1692645394)